src/Raven.Server/Smuggler/DatabaseDataImporter.cs

```diff
@@ -237,6 +220,44 @@ public DatabaseDataImporter(DocumentDatabase database)
             return result;
         }
 
+        private async Task FinishBatchOfDocuments()
+        {
+            if (_prevCommand != null)
```

Comment:
There is code duplication here, you can just call HandleBatchOfDocuments() instead